app/models/manifest_uploader.rb

```diff
@@ -45,9 +50,53 @@ def missing_locations
     @missing_locations ||= location_barcodes.reject { |barcode| locations.key?(barcode.strip) }
   end
 
+  def ordered_location_barcodes
+    @ordered_location_barcodes ||= location_barcodes.select { |barcode| locations[barcode.strip].type == "OrderedLocation" unless locations[barcode.strip].nil? }
```

Comment:
Maybe `locations.include?(barcode.strip)` is nicer? Think that would do the same?

Reply:
Yep that is nicer - I had to switch the `unless` for an `if` to make the logic clearer